src/ScrollView/scrollView.scss

```diff
@@ -0,0 +1,113 @@
+@import '~gemini-scrollbar/gemini-scrollbar.css';
+@import '~sass-flex-mixins/src/sass-flex-mixins.scss';
+
+$scrollbarSize: 7px !default;
+$scrollbarSizeBig: 9px !default;
```

Comment:
do you think we need a variable? couldn't we dynamically define as `$scrollbar-size + 123px`?